pocket_coffea/workflows/tthbb_base_processor.py

```diff
@@ -44,7 +44,7 @@ def apply_object_preselection(self, variation):
             self.events, "Jet", self.params, "LeptonGood"
         )
         self.events["BJetGood"] = btagging(
-            self.events["JetGood"], self.params.btagging.working_point[self._year]
+            self.events["JetGood"], self.params.btagging.working_point[self._year], wp="M"
```

Comment:
I would keep the chosen working point as part of the configuration of the analysis and not as part of the processor.  
This may fix the `object_preselection` part of the params within the jet dictionary

Reply:
Agree, maybe we can do it by passing the b-tag working point in the `workflow_options` in the Configurator?
If not passed, we can keep the medium working point as default.

Reply:
In my opinion it is not a workflow option, but a preselection params like others, so to be put here: https://github.com/PocketCoffea/AnalysisConfigs/blob/main/configs/ttHbb/semileptonic/ttbar_background/params/object_preselection_semileptonic.yaml for example.

Reply:
Makes sense. Should we add an entry to the `Jet` key to specify the btag WP?
I was thinking something like:
```
Jet:
    ...
    btag:
        wp: M
```
and this key is used to define the collection `BJetGood` in the object preselection of custom processors.

Instead, the b-tagging algorithm, the working points and SF can stay in the central parameters in https://github.com/PocketCoffea/PocketCoffea/blob/337ef90fbba299be4a2c75586a4808fd9c1ed950/pocket_coffea/parameters/btagging.yaml

Reply:
sure. The compatibility of the configuration for the preselection and the function is there if one follows the default structure.   

If a user needs more complex selections or configuration both the function and the parameters can be customized. I think this is a sensible default

Reply:
Update: now the b-tagging WP is read from the object preselection parameters [here](https://github.com/PocketCoffea/PocketCoffea/blob/5fa68398e5c02892ea9f1e67fd0a2a395a974ac2/pocket_coffea/workflows/tthbb_base_processor.py#L47).
However the structure of the dictionary `self.params.object_preselection.Jet["btag"]` is not enforced anywhere and it's up to the user how to pass the b-tagging WP.